doc/user/content/sql/drop-type.md

```diff
@@ -17,30 +17,88 @@ menu:
 Field | Use
 ------|-----
 _data_type_name_ | The name of the type to remove.
-`CASCADE` | Automatically drop objects that depend on the type (such as table columns, functions, operators, or other types)
+`CASCADE` | Automatically removes objects that depend on the type, such as tables or other types.
 `IF EXISTS`  |  Issue a notice, but not an error, if the type doesn't exist.
-`RESTRICT`  |  Refuse to drop the type if any objects depend on it. This is the default.
+`RESTRICT`  |  Refuse to remove the type if any objects depend on it. This is the default.
 
 
 ## Examples
 
-These examples use the custom drop types from the [CREATE TYPE examples](../create-type#examples).
+### Remove a data type with no dependencies
+```sql
+CREATE TYPE int4_map AS MAP (key_type=text, value_type=int4);
 
-### DROP TYPE (no dependencies)
+SHOW TYPES;
+```
+```
+CREATE TYPE
+    name
+--------------
+  int4_map
+(1 row)
+```
 
 ```sql
-DROP TYPE int4_list_list;
+DROP TYPE int4_map;
+SHOW TYPES;
 ```
 ```
 DROP TYPE
+  name
+--------------
+(0 rows)
 ```
 
+### Remove a data type with dependent objects
 
-### DROP TYPE (dependencies)
+{{< warning >}}
+By default, `DROP TYPE` will not remove a type with dependent objects. The `CASCADE` switch will remove both the specified type and *all its dependent objects*.
+{{< /warning >}}
+
+In the example below, the `CASCADE` switch removes `int4_list`, `int4_list_list` (which depends on `int4_list`), and the table *t* which has a column of data type `int4_list`.
 
 ```sql
-DROP TYPE _int4_map_ CASCADE;
+CREATE TYPE int4_list AS LIST (element_type = int4);
+
+CREATE TYPE int4_list_list AS LIST (element_type = int4_list);
+
+CREATE TABLE t (a int4_list);
+
+SHOW TYPES;
+```
+```
+CREATE TYPE
```

Comment:
What command generates this output?

Reply:
Sorry, copy & paste error. I've corrected it to:

`CREATE TYPE
CREATE TYPE
CREATE TABLE
      name      
----------------
 int4_list
 int4_list_list
(2 rows)`